code/modules/clothing/suits/utility.dm

```diff
@@ -94,14 +94,19 @@
 
 
 /obj/item/clothing/head/bomb_hood/security
+	name = "security bomb hood"
+	desc = "Use in case of bomb. This one has improved armor and a snazzy red finish."
 	icon_state = "bombsuit_sec"
 	item_state = "bombsuit_sec"
+	armor = list("melee" = 35, "bullet" = 30, "laser" = 30,"energy" = 10, "bomb" = 100, "bio" = 0, "rad" = 0, "fire" = 80, "acid" = 50)
 
 /obj/item/clothing/suit/bomb_suit/security
+	name = "security bomb suit"
+	desc = "A suit designed for safety when handling explosives. This one has improved armor and a snazzy red finish."
 	icon_state = "bombsuit_sec"
 	item_state = "bombsuit_sec"
-	allowed = list(/obj/item/gun/energy, /obj/item/melee/baton, /obj/item/restraints/handcuffs)
-
+	allowed = list(/obj/item/gun/energy, /obj/item/melee/baton, /obj/item/restraints/handcuffs, /obj/item/device/transfer_valve, /obj/item/gun/ballistic/shotgun)
+	armor = list("melee" = 40, "bullet" = 35, "laser" = 35,"energy" = 10, "bomb" = 100, "bio" = 0, "rad" = 0, "fire" = 80, "acid" = 50)

```

Comment:
These values are WAY too high. a bomb suit is not supposed to be able to replace security gear.

Reply:
Its weaker than the actual bulletproof vest for bullet damage and weaker than riot gear for melee while also making you incredibly slow.

Reply:
isn't this literally a few points higher than the regular bomb suit

Reply:
????
/obj/item/clothing/suit/armor/vest
is armor = list(melee = 30, bullet = 30, laser = 30, energy = 10, bomb = 25, bio = 0, rad = 0

Reply:
even checked it was the right one 
/obj/structure/closet/secure_closet/security/PopulateContents()
	..()
	new /obj/item/clothing/suit/armor/vest(src)

Reply:
Hos armor doesn't make you move slower than a slug